ros/src/waypoint_updater/nodes/updater.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+
+import rospy
+import math
+
+from geometry_msgs.msg import PoseStamped, TwistStamped
+from styx_msgs.msg import Lane, Waypoint
+from std_msgs.msg import String
+from dynamic_reconfigure.server import Server
+from waypoint_updater.cfg import DynReconfConfig
+
+
+'''
+This node will publish waypoints from the car's current position to some `x`
+distance ahead.  As mentioned in the doc, you should ideally first implement
+a version which does not care about traffic lights or obstacles.
+
+Once you have created dbw_node, you will update this node to use the status of
+traffic lights too.
+
+Please note that our simulator also provides the exact location of traffic
+lights and their current status in `/vehicle/traffic_lights` message. You
+can use this message to build this node as well as to verify your TL
+classifier.
+
+'''
+
+
+class WaypointUpdater(object):
+    def __init__(self):
+        self.dyn_vals_received = False
+        self.waypoints = []
+        self.pose = None
+        self.velocity = None
+        self.final_waypoints = []
+        self.next_traffic_light_wp = None
+        if not self.dyn_vals_received:
+            self.update_rate = 3
+            self.default_velocity = 0.0
+            self.lookahead_wps = 50
+        self.subs = {}
```

Comment:
Can we set the init parameter defaults to the same values as in the .cfg file?

Reply:
I will put them to the same values.  I had set them this way when testing to observe they were changed properly at startup from dynamic_reconfigure - since explicit logging only happens when the values are changed.